.github/workflows/release.yaml

```diff
@@ -2,33 +2,51 @@ name: release
 
 on:
   push:
-    tags: "v*"
+    tags:
+      - "v*"
 
 # Reduce default permission of GITHUB_TOKEN to nothing
 # Repository can still be checked out during these jobs
 permissions: {}
 
 jobs:
+  context:
+    runs-on: ubuntu-latest
+    outputs:
+      cosign_version: ${{ steps.get_context.outputs.cosign_version }}
+      chart_version: ${{ steps.get_context.outputs.chart_version }}
+      original_repo: ${{ steps.get_context.outputs.original_repo }}
+      original_tag: ${{ steps.get_context.outputs.original_tag }}
+      original_image: ${{ steps.get_context.outputs.original_image }}
+      build_repo: ${{ steps.get_context.outputs.build_repo }}
+      build_tag: ${{ steps.get_context.outputs.build_tag }}
+      build_image: ${{ steps.get_context.outputs.build_image }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Get context
+        id: get_context
+        uses: ./.github/actions/context
+
   version-match:
     runs-on: ubuntu-latest
+    needs: [context]
     steps:
       - uses: actions/checkout@v3
-      - name: Install yq
-        run: sudo snap install yq
       - name: Ensure version equality
         run: |
-          IMAGE_TAG=$(yq e '.deployment.image' helm/values.yaml| cut -d ':' -f2)
+          IMAGE_TAG=${{ needs.context.outputs.original_tag }}
           for COMMIT_TAG in $(git tag --points-at $(git rev-parse HEAD)); do
             if [[ "$IMAGE_TAG" == "$COMMIT_TAG" ]]; then
               exit 0
             fi
           done
           echo "Tag '$IMAGE_TAG' is not within tags of commit: $(git tag --points-at $(git rev-parse HEAD))"
-          exit 1
```

Comment:
Why change this? Seems like a guardrail that was put in place for a reason

Reply:
yeap...that was by accident as is being fixed. Good catch